### PR TITLE
Kalman filter alert system project documentation

### PR DESCRIPTION
**Pull request by economuteo**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by...

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: 
Fotescu Maria-Irina fotescufmi03@yahoo.com
Signed-off-by: 
Economu Teo-Antonio teoantonioeconomu@gmail.com

### CODE REVIEW

File: docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+[2:22 PM] Maria-Irina FOTESCU (135354)
```
Comment (alexandruradovici) [inline docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:1]:
```suggestion
```

File: docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+The format is
+
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+
+-->
+
+| Device                                                                                                  | Usage               | Price                                                                                                                                                                                                                                                                                |
+| ------------------------------------------------------------------------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```
Comment (alexandruradovici) [inline docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:55]:
```suggestion
|-|-|-|
```

File: docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+| [Rapspberry Pi Pico](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [39 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html)                                                                                                                                                                                        |
+| [Breadboard]()                                                                                          | Main board          | [0 RON(Already had this)](https://www.emag.ro/kit-plusivo-microcontroller-starter-programabil-in-arduino-ide-x001fpqyl1/pd/DKJN9VMBM/?utm_source=mobile%20app&utm_medium=ios&utm_campaign=share%20product)                                                                           |
```
Comment (alexandruradovici) [inline docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:57]: This link is broken.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template and add two *Signed-off by* lines.

File: docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+# Kalman Temperature Alert System
+
```
Comment (alexandruradovici) [inline docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:2]: Please add a one line description of your project.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

File: docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md (hunk 4 above)
Comment (economuteo) [reply docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:2]: Ok!

Comment (alexandruradovici) [reply docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:2]: Not added.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template (description) signing off both of your names and emails.

File: docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md (hunk 4 above)
Comment (economuteo) [reply docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md:2]: Is it ok now?

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**